maxim/test_runs/run_utils.py

```
@@ -1,9 +1,9 @@
 import asyncio
-from typing import Awaitable, List, Optional
+from typing import Awaitable, List, Optional, Dict
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Awaitable`

Import from `collections.abc`

(UP035)